### ISSUE TITLE
Add encryption to backup

### ISSUE THREAD

**Issue by fradev-old** [pothi/backup-wordpress#18]
Hello Pothi,
First of all I would like to thank you since I've found your project really useful.
I live and work in Europe so cause of the new GDPR rules (a law that enforces privacy and data processing) I've forked your work and added the option to encrypt(simple encryption using gpg) backup before sending it to aws.
I don't know if it might be interesting for you, if so here the commit link: https://github.com/Cicciodev/backup-wordpress/commit/cb18f5e4c241761f0571e896e76c0e2a76b8f6c2

Have a nice day

**Comment by pothi** [comment]
Hi @Cicciodev ,

Thank you for enlightening me on GDPR.

That's a nice addition considering how important it is to be complaint with GDPR. I tested your changes. All seem good.

Can you please send a pull request so that you are credited for your work?

**Comment by pothi** [comment]
Hi @Cicciodev ,

I went on and created a pull request from your repo.

I will fine-tune it from here such as migrating your changes to other scripts (such as db-backup.sh) and then asking the user to change the default passphrase (if the existing passphrase is not modified).

Thanks again,
Pothi.

**Comment by Cicciodev** [comment]
Hi @pothi ,
I'm glad that you found it useful. 
Sorry I've modified only the full_site script, I'm not so good in sheel scripting and so I've tried to capy as much as possible from your coding style. 

> I will fine-tune it from here such as migrating your changes to other scripts (such as db-backup.sh) and then asking the user to change the default passphrase (if the existing passphrase is not modified).

You're right I was planning to leave it empty so that if there is a passphrase the script will load the encrypted backup otherwise the non-crypted backup, I thought that it would be better to leave to the user the coiche.

Btw I was planning also to estend your script to platform suche as Moodle and Limesurvey, I'll do it next week.

thanks to you,
Francesco
